cheroot/test/test_core.py

```diff
@@ -17,20 +17,24 @@
 
 
 localDir = os.path.dirname(__file__)
-base_path = os.path.join(os.getcwd(), localDir)
-favicon_path = os.path.join(base_path, '../favicon.ico')
+temp_dir = os.path.join(os.getcwd(), localDir, 'temp')
```

Comment:
Why did you rename it? It's used to be `static` since times of being in `cherrypy` repo's tests. I believe there are other tests which still rely on `static` folder.